generators/ci-cd/templates/.gitlab-ci.yml.ejs

```diff
@@ -152,18 +152,19 @@ before_script:
     <%_ } _%>
     - export MAVEN_USER_HOME=`pwd`/.maven
     <%_ if (!skipClient) { _%>
+    # enable next lines if you want to install frontend tool over maven
```

Comment:
I think we can't disable at in general. In case we are not inside a docker environment the required tools are not installed (most likely). So I would suggest we need to guard that. See

```
<%_ if (insideDocker) { _%>
image: jhipster/jhipster:v<%= jhipsterVersion %>
<%_ } _%>
```

Reply:
Yes, the improvement comes because npm install is executed only once and also the tools e.g. node, yarn, npm are not getting installed over maven. A good tip with catching up if is inside docker or not. I will update my PR.